migration-guide.md

```diff
@@ -17,3 +17,4 @@ Migration from 0.5.x/0.6.x to 1.0.x
     }
     ```
 * If you were using the scala serializers independently adapt imports from `com.romix.scala.serialization.kryo` to `io.altoo.akka.serialization.kryo.serializer.scala`
+* Configuration property `id-strategy` has been re-named to `id-strategy` and the default has been changed from `incremental` to `default`.
```

Comment:
add: please read the documentation (link to reference.conf) for the different strategies and the implications.